converters/fromZigbee.js

```diff
@@ -4726,6 +4726,18 @@ const converters = {
             });
         },
     },
+    tuya_data_point_dump_alt: {
```

Comment:
It was intended to enable `tuya_data_point_dump_alt` on my local installation for debugging, but of course I can enable it for everyone. Pls. find next commit below.

Reply:
Could we maybe already integrate `tuya_data_point_dump_alt` into  `tuya_data_point_dump`?

Reply:
Not now, because it needs debugging to be sure if decoding received data in `cluster.ts` `activeStatusReportAlt` is really decodeable with `uint16 `and `BuffaloZclDataType`
```
                    {name: 'seq', type: DataType.uint16},
                    {name: 'dpValues', type: BuffaloZclDataType.LIST_TUYA_DATAPOINT_VALUES},
```
My intention from the beginning was to add **minimal code changes** to
- support new 0x05 command in `cluster.ts` `activeStatusReportAlt`
- add `activeStatusReportAlt` to `fromZigbee ``tuya_data_point_dump_alt`
- simply to be able to proceed with debugging on my local system with `tuya_data_point_dump_alt`
- in order to prepare production-ready code changes with a future PR

I did not want to break anything existing, thats why I did not at once add
- `activeStatusReportAlt` to existing `fromZigbee` `tuya_data_point_dump`
- `activeStatusReportAlt` to existing `fromZigbee` `ZB006X_settings`
- `fz.tuya_data_point_dump_alt` to existing `devices/fantem.js` 
- and instead copied `tuya_data_point_dump` into `tuya_data_point_dump_alt`

My goal was just beeing able to debug, proof my local code is working fine and **afterwards**
- `activeStatusReportAlt` to existing `fromZigbee` `tuya_data_point_dump`
- `activeStatusReportAlt` to existing `fromZigbee` `ZB006X_settings`
- remove `fromZigbee ``tuya_data_point_dump_alt` again.

If you show me a way to add 0x05 command in `cluster.ts` `activeStatusReportAlt` by changes to my local filesystem, I will stop asking for PRs.

Reply:
I've merged https://github.com/Koenkk/zigbee-herdsman/pull/519 now, so you can keep the changes in this PR in your local external converter. Once production code is ready you can update this PR.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)